packages/manager/modules/nutanix/src/dashboard/routing.js

```diff
@@ -19,6 +19,8 @@ export default /* @ngInject */ ($stateProvider) => {
         NutanixService.getServiceDetails(serviceInfo.serviceId),
       nutanixPlans: /* @ngInject */ (user, NutanixService) =>
         NutanixService.getNutanixPlans(user.ovhSubsidiary),
+      newClusterType: /* @ngInject */ (NutanixService, serviceInfo) =>
+        NutanixService.isNewCluster(serviceInfo.serviceId),
```

Comment:
**nitpick(perf):** serviceDetails are already fetched on `serviceDetails`. So this additional call is redundant and can be avoided. 
```suggestion
      newClusterType: /* @ngInject */ (serviceDetails) =>
        NEW_CLUSTER_PLAN_CODE.includes(serviceDetails.billing.plan.code),
```

Reply:
Hi Anoop, Its not same api call. ServiceDetails calls `/service/${serviceId}` where has I need to call `/services/${serviceId}`.  I will make use of getServicesDetails in service.